pkg/util/validation.go

```diff
@@ -18,19 +18,32 @@ package util
 
 import (
 	"regexp"
-	"strings"
 )
 
-// IsDNSLabel tests for a string that conforms to the definition of a label in
-// DNS (RFC 1123).
-func IsDNSLabel(value string) bool {
-	return IsDNS1123Label(value)
+const LabelValueFmt string = "[-A-Za-z0-9_.]*"
```

Comment:
Oh, one thing - qualified name currently cannot either start or end with punctuation. Is this constraint OK for label values as well?